app/helpers/observations_helper.rb

```diff
@@ -125,9 +121,7 @@ def observation_naming_buttons(observation, do_suggestions)
     buttons.safe_join(tag.br)
   end
 
-  private
```

Comment:
Why did you entirely get rid of private from this?  I'm not seeing any external references.

Reply:
@mo-nathan - See the commit comment. Helpers cannot have private methods.

Reply:
Oh, I see the link now.  Reading it over, it doesn't seem to say that they can't be private, just that they get mixed in so it doesn't have as big an effect as you might expect.  However, it would still be private for things that try to access whatever is loading the helper.  In my view the biggest reason to make things private is as a declaration of what the author of the code expects other folks to use and what what they shouldn't.  There are ways of calling private methods if someone is really determined to do.  The key difference to me is if they do, then if you remove or change the interface it's on them.

Reply:
OK, thank you for reading through that discussion. I didn't get how any other context could expose a private method, but i guess if it's mixed into the calling class, then... that sort of makes sense. Still — I had no idea I needed to be aware of namespace collisions in separate helpers!

I have been declaring tons of methods private throughout these CRUD refactors, I definitely prefer the convention. Will restore the `private` here.

Reply:
Restored private methods